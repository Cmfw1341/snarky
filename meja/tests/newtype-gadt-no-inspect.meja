type t('a) = A(int) : t(int) | B(bool) : t(bool);

let f = fun (type t, x : t) => {
  switch (x) {
    | A(_) => 15
    | B(_) => 20
  };
};
