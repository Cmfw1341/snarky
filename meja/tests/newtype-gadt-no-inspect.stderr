File "tests/newtype-gadt-no-inspect.meja", line 5, characters 6-10:
Error: Cannot unify _ and t(int)
